compiler/rustc_typeck/src/check/method/confirm.rs

```diff
@@ -148,16 +151,24 @@ impl<'a, 'tcx> ConfirmContext<'a, 'tcx> {
                 );
             }
         };
+
+        debug!("derefd_ty={:#?}", derefd_ty);
```

Comment:
This can just be 
```suggestion
        debug!(?derefd_ty);
```

today

Reply:
I didn't know this syntax, thanks :)